### ISSUE TITLE
[Feature Request]: Add Import/Export feature for posts to facilitate collaboration with other CMS

### ISSUE THREAD

**Issue by s3nnet** [GetPublii/Publii#1179]
### Feature Description

Open-source software is all about collaboration.
An import/export feature for sharing articles would be awesome!

**Comment by thefourcraft** [comment]
Yep, this is a fantastic idea! I would love to be able to use my blog on different machines  (eg one PC and other PC )
